### ISSUE TITLE
Extent not getting set on Map to what I set it to

### ISSUE THREAD

**Issue by gisuser1024** [mapnik/mapnik#3471]
New to mapnik so might not be doing things correctly.  
I'm using node-mapnik as sort of a custom WMS Server. (returning a PNG file to a client using Open Layers).    The WMS server is basically just returning a layer of Points.  
I'm creating the map with pixel of 2048X2048 and also creating an mapnik.Image of the same pixels.

My buffersize = 0.
I've tried with both aspect_fix_mode = 0 and 1.  

My srs = +init=epsg:4326

I'm noticing that if I do not set an extent on the map it gives me like 
1.79.....e+308,
1.79.....e+308,
-1.79.....e+308,
-1.79.....e+308,

I'm basically taking the bounding box of what OpenLayers passes in and trying to set the extent of this on the map.   

When I have an extent coming back from Open layers (3). such as  (minlon,minlat,maxlon,maxlat)
[30,29,56,39]

I can see from printing the attributes of the map that it gets set to 
something such as
[30,21,56,47]

Basically it's clipping my latitude (pretty significant).   So when I am viewing my map through my browser (using Open Layers).   The image is clipped on the top and on the bottom and not showing me any points.  

Do you know why my extent is not getting set to what I specify.  I feel like it's related to height/width of the image but not sure.
